src/book.py

```diff
@@ -0,0 +1,96 @@
+import uuid
+import os
+import logging
+from datetime import datetime
+
+from flask import Blueprint, jsonify, request
+from cloudinary.uploader import upload
+from cloudinary.utils import cloudinary_url
+
+from src.models import Author, Book,  db
+from src.google import get_user_info
+from src.constants.http_status_codes import HTTP_201_CREATED, HTTP_400_BAD_REQUEST
+
+book_bp = Blueprint('book', __name__, url_prefix='/api')
+
+
+@book_bp.post('/books')
+def create_book():
+    file = request.files['image']
+    if not file:
+        return jsonify({
+            'error': "book image missing"
+        }), HTTP_400_BAD_REQUEST
+
+
+    # Author
+    book_author = request.form['author'].split(' ')
+    author_first_name = book_author[0]
+    author_last_name = ' '.join(book_author[1:])
```

Comment:
Err, would be better if we get these values individually instead of having a split process here. first name and last name should be gotten just like title and the rest.

Reply:
This has been separated.